accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/RestrictionMapsHolder.java

```diff
@@ -0,0 +1,47 @@
+package biz.netcentric.cq.tools.actool.helper;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.jcr.Value;
+/**
+ * Container class holding 2 maps for storing single and multivalue restrictions. singleValuedRestrictionsMap holds restriction names as keys and values of type javax.jcr.Value
+ * multiValuedRestrictionsMap holds restriction names as keys and values of type javax.jcr.Value[]
+ * @author jochenkoschorkej
+ *
+ */
+public class RestrictionMapsHolder {
+    private final static Map<String, Value> emptySingleValuedRestrictionsMap = Collections.unmodifiableMap(new HashMap<String, Value>());
+    private final static Map<String, Value[]> emptyMultiValuedRestrictionsMap = Collections.unmodifiableMap(new HashMap<String, Value[]>());
```

Comment:
Why can't they be static as well? If I make them instance variables, I cannot use them in static EMPTY map